mail/templates/email_base.html

```diff
@@ -238,8 +238,7 @@
 	            <tr>
 	                <td style="padding: 20px;">
                       {% block logo %}
-                      <img src="{{ base_url }}/static/images/mit-logo.jpg" width="78" height="41" alt="{{ site_name }}" style="height: auto; background: #ffffff; font-family: sans-serif; font-size: 15px; line-height: 15px; color: #A31F34; margin-bottom: -10px;  margin-right: 10px;""/>
-                      <span style="color: #A31F34; font-family: sans-serif; font-size: 28px; line-height: 28px;">{{ site_name }}</span>
+                      <img src="{{ base_url }}/static/images/mitx-pro-logo.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; background: #ffffff; font-family: sans-serif; font-size: 15px; line-height: 15px; color: #A31F34; margin-bottom: -10px;  margin-right: 10px;"/>
```

Comment:
Is there a specific reason we are using `base_url` instead of `{% static ...`?

Reply:
You can't as it has to be render in a email. I have tested this on my local.